requirements-dev.txt

```diff
@@ -0,0 +1,7 @@
+bitarray==2.7.3
```

Comment:
bitarray -> unicodecsv should only be present in requirements.txt and not here.
In requirements-dev.txt we only have the extra dependencies that would be present in addition to the normal dependencies, in developement configuration.

Reply:
Got it. Removed.